.github/workflows/goreleaser.yml

```diff
@@ -0,0 +1,9 @@
+on:
```

Comment:
![image](https://github.com/user-attachments/assets/ada021c5-dbd3-4486-82e9-86c34e65d63e)
I'm not sure how it would look if the workflow doesn't have a name here

Reply:
Good catch!